Support/ceptr/tests/test_regression.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
Can you remove this file from the PR? It's a great idea but I think best done as a separate thing.

Reply:
I removed it from the ceptr-fix branch.